packages/rum-recorder/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -153,7 +153,7 @@ export function doStartSegmentCollection(
     state = {
       status: SegmentCollectionStatus.SegmentPending,
       segment,
-      expirationTimeoutId: setTimeout(
+      expirationTimeoutId: window.setTimeout(
```

Comment:
This shouldn't be needed. We'll have to audit which files are included during e2e tests and avoid including too many files in nodejs.